pkg/management/endpoint.go

```diff
@@ -440,6 +441,15 @@ func MakeClearUserLoginFailures(component Component) cs.Endpoint {
 	}
 }
 
+// MakeGetAttackDetectionStatus creates an endpoint for GetAttackDetectionStatus
+func MakeGetAttackDetectionStatus(component Component) cs.Endpoint {
+	return func(ctx context.Context, req interface{}) (interface{}, error) {
+		var m = req.(map[string]string)
+
```

Comment:
credentialID is in the body?
or maybe there is no credentialID

Reply:
credentialID was an error in the swagger file.